### PR TITLE
TAJO-1803: Use in-memory derby as the default catalog for unit tests

### PR DESCRIPTION
**Pull request by jihoonson**
[Review on Reviewable](https://reviewable.io/reviews/apache/tajo/714)

### CODE REVIEW

**Comment by jihoonson** [comment]
I've also fixed some bugs in catalog.

**Comment by jihoonson** [comment]
This patch is ready for review.

File: tajo-core-tests/src/test/java/org/apache/tajo/engine/query/TestSelectQuery.java [hunk 1]
```diff
@@ -26,9 +26,11 @@
 import org.apache.tajo.TajoConstants;
 import org.apache.tajo.TajoProtos.QueryState;
 import org.apache.tajo.TajoTestingCluster;
+import org.apache.tajo.catalog.CatalogConstants;
```
Comment (hyunsik) [inline tajo-core-tests/src/test/java/org/apache/tajo/engine/query/TestSelectQuery.java]: unused imports

File: tajo-common/src/main/proto/errors.proto [hunk 2]
```diff
@@ -157,6 +157,9 @@ enum ResultCode {
   CAT_UPGRADE_REQUIRED                  = 901;  // Migration
   CAT_CANNOT_CONNECT                    = 902;  // Cannot connect metadata server
 
+  // Catalog store
+  UNSUPPORTED_CATALOG_STORE             = 905; // Unsupported type of catalog store
```
Comment (hyunsik) [inline tajo-common/src/main/proto/errors.proto]: It seems to be necessary the prefix ``CAT``.

**Comment by hyunsik** [comment]
I used some trivial comments. You can commit the patch after you reflect them if you agree with my comments.

**Comment by jinossy** [comment]
+1 LGTM

**Comment by jihoonson** [comment]
Thanks for quick review!

**Pull request closed without merging**